test/end-to-end/offer.js

```diff
@@ -698,6 +698,42 @@ describe("Offer endpoint tests", () => {
             });
         });
 
+        describe("Same 'publishDate' and 'publishEndDate'", () => {
```

Comment:
you can have a beforeAll block where you delete all Offers. Then you should use generateTestOffer util below